src/bundle/Resources/public/js/scripts/helpers/table.helper.js

```diff
@@ -0,0 +1,20 @@
+(function (global, doc, eZ) {
+    const onChangeHandler = (event) => {
+        const { activeClass, checked } = event.target;
+        const action = checked ? 'add' : 'remove';
+        const parentRow = event.target.closest('tr');
+
+        parentRow.classList[action](activeClass);
+    };
+
+    const parseCheckbox = (checkboxSelector, activeClass) => {
+        doc.querySelectorAll(checkboxSelector).forEach((checkboxNode) => {
+            checkboxNode.activeClass = activeClass;
```

Comment:
Please do not extend browsers object with custom propeties, you should use data attribute instead in such case.
But in this case the best aproach would be to use bind in `onChangeHandler.bind(this, activeClass)`